docs/0.10.0/html/_sources/profiler.rst.txt

```diff
@@ -0,0 +1,955 @@
+* null_replication_metrics - statistics of data partitioned based on whether column value is null (index 1 of lists referenced by dict keys) or not (index 0)
+    * class_prior - a list containing probability of a column value being null and not null
+    * class_sum - a list containing sum of all other rows based on whether column value is null or not
+    * class_mean - a list containing mean of all other rows based on whether column value is null or not
+
+Unstructured Profile
+~~~~~~~~~~~~~~~~~~~~
+
+**global_stats**:
+
+* samples_used - number of input data samples used to generate this profile
+* empty_line_count - the number of empty lines in the input data
+* file_type - the file type of the input data (ex: .txt)
+* encoding - file encoding of the input data file (ex: UTF-8)
+* memory_size - size of the input data in MB
+* times - duration of time it took to generate this profile in milliseconds
+
+**data_stats**:
+
+* data_label - labels and statistics on the labels of the input data
+    * entity_counts - the number of times a specific label or entity appears inside the input data
+        * word_level - the number of words counted within each label or entity
+        * true_char_level - the number of characters counted within each label or entity as determined by the model
+        * postprocess_char_level - the number of characters counted within each label or entity as determined by the postprocessor
+    * entity_percentages - the percentages of each label or entity within the input data
+        * word_level - the percentage of words in the input data that are contained within each label or entity
+        * true_char_level - the percentage of characters in the input data that are contained within each label or entity as determined by the model
+        * postprocess_char_level - the percentage of characters in the input data that are contained within each label or entity as determined by the postprocessor
+    * times - the duration of time it took for the data labeler to predict on the data
+* statistics - statistics of the input data
+    * vocab - a list of each character in the input data
+    * vocab_count - the number of occurrences of each distinct character in the input data
+    * words - a list of each word in the input data
+    * word_count - the number of occurrences of each distinct word in the input data
+    * times - the duration of time it took to generate the vocab and words statistics in milliseconds
+
+Graph Profile
+~~~~~~~~~~~~~~~~~~
+
+* num_nodes - number of nodes in the graph
+* num_edges - number of edges in the graph
+* categorical_attributes - list of categorical edge attributes
+* continuous_attributes - list of continuous edge attributes
+* avg_node_degree - average degree of nodes in the graph
+* global_max_component_size: size of the global max component
+
+**continuous_distribution**:
+
+* <attribute_N>: name of N-th edge attribute in list of attributes
```

Comment:
Is this the right formatting?

Reply:
looks right when I look at the docs https://taylorfturner.github.io/DataProfiler/docs/0.10.0/html/profiler.html?highlight=attribute_n#:~:text=continuous_distribution